src/wizard/javascript/react.md

```diff
@@ -15,39 +21,39 @@ yarn add @sentry/react @sentry/tracing
 npm install --save @sentry/react @sentry/tracing
 ```
 
-Next, import and initialize the Sentry module as early as possible, before initializing React:
+## Configure
+Initialise Sentry as early as possible in your application's lifecycle.
```

Comment:
```suggestion
Initialize Sentry as early as possible in your application's lifecycle.
```
Same comment as before - I'm wondering if "application's lifecycle" is a bit vague? Maybe it's just vague to me, but I'm wondering if there's a way to call this something more specific?